Sources/Basic/Lock.swift

```diff
@@ -58,8 +58,8 @@ public final class FileLock {
     /// Note: This method can throw if underlying POSIX methods fail.
     public func lock() throws {
       #if os(Windows)
-        if h == nil {
-            let h = lockFile.pathString.withCString(encodedAs: UTF16.self, {
+        if handle == nil {
+          let h = lockFile.pathString.withCString(encodedAs: UTF16.self, {
```

Comment:
Thanks! Minor thing: can you fix the indentation here?

Reply:
Yeah, sure! ;)